README.md

```diff
@@ -76,6 +72,14 @@ tmt \
             report -h html
 ```
 
+## Waiving failed results
+
+In this context, "to waive" means to discard a failing result as known-bad,
```

Comment:
`to discard` is not correct wording. Results won't be discarded, they are not going to cause overall fail, but still will be shown as WARN.

Reply:
This is a high-level description from an overall process workflow PoV. "To discard" as in "to render uninteresting". Maybe I can try rephrasing it differently.